install/prepare_site_init.md

```diff
@@ -332,6 +332,12 @@ with system-specific customizations.
        yq read "${SITE_INIT}/customizations.yaml" spec.kubernetes.services.cray-keycloak-users-localize
        ```
 
+    1. (Optional) Configure IPv6 support for the `cray-keycloak` service.
+
+       See the [IPv6 Configuration Guide](../operations/network/customer_accessible_networks/ipv6_configuration_guide.md#keycloak) for more information.
+
+       > **Important** Do not perform this step without first configuring the `cmn-cidr6`, `chn-cidr6`, `cmn-gateway6`, and `chn-gateway6` settings in `system_config.yaml`
```

Comment:
Isn't `system_config.yaml` edited and used earlier in the install process than this? I'll double check. If so, we also need to add a similar step at that point, so they don't get this far into it and then have to backtrack.